LeetcodeProblemsTests/Algorithms/Time_Needed_Rearrange_Binary_String_Test.js

```diff
@@ -0,0 +1,9 @@
+const assert = require('assert');
+const secondsToRemoveOccurrences = require('../../LeetcodeProblems/Algorithms/Time_Needed_Rearrange_Binary_String').secondsToRemoveOccurrences;
+
+function test() {
+  assert.equal(secondsToRemoveOccurrences("0110101"));
+  assert.equal(secondsToRemoveOccurrences("11100"))
```

Comment:
could you add the assert equals to a number?

Reply:
i have added assertion, also i saw there was assertion missing for my last PR added that change as well. 

also this is  O ( N ) solution

Reply:
The `replaceAll` would have to iterate over the whole string, and the while(true) iterates over it too (it is not fixed 🤔 ).